community/php7-rrd/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Pascal Hofmann <mail@pascalhofmann.de>
+# Maintainer: Pascal Hofmann <mail@pascalhofmann.de>
+pkgname=php7-rrd
+_pkgreal=rrd
+pkgver=2.0.1
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP$_phpver bindings to rrd tool system"
+url="https://pecl.php.net/package/rrd"
+arch="all"
+license="BSD"
+depends="php$_phpver-common php$_phpver-session"
+makedepends="php$_phpver-dev autoconf rrdtool-dev"
+source="$pkgname-$pkgver.tgz::http://pecl.php.net/get/$_pkgreal-$pkgver.tgz"
+options="!check"
```

Comment:
Could you add a comment here on the reason there's no check function? E.g. no upstream test suite, or it being unfixably broken, et cetera.